lib/child_process.js

```diff
@@ -258,26 +251,20 @@ exports.execFile = function(file /*, args, options, callback*/) {
         if (!encoding)
           _stdout.push(chunk);
         else
-          _stdout += chunk;
+          _stdout += chunk.toString(encoding);
```

Comment:
If we're going to be pushing chunks always for `stderr` we need to do the same for `stdout`.